communicator/mesoshttp/mesosapi.go

```diff
@@ -0,0 +1,73 @@
+package mesoshttp
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/vmturbo/mesosturbo/communicator/metadata"
+	"github.com/vmturbo/mesosturbo/communicator/util"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type MesosHTTPClient struct {
+	MesosMasterBase string
+}
+
+func (mesos *MesosHTTPClient) MesosPostRequest(metadata *metadata.ConnectionClient, dcos_token string) error {
+
+	var jsonStr []byte
+	url := "http://" + metadata.MesosIP + "/acs/api/v1/auth/login"
```

Comment:
The name of the method is MesosPostRequest, but the url is hardcoded to a very specific operation. Please either change the name of the method, or make the url as an argument of the method.